apps/website/src/app/learn/page.tsx

```diff
@@ -290,17 +290,16 @@ await verifyProof(verificationKey, fullProof)`,
                     mt={{ base: "100px", md: "170px" }}
                     mb={{ base: "50px", md: "112px" }}
                 >
-                    <Box overflow="auto" mx="3">
-                        <TabList gap="40px" w="max-content" whiteSpace="nowrap">
-                            <Tab px={0} fontSize="24px">
+                    <Box mx="3" overflowX={{ base: "auto", md: "hidden" }}>
+                        <TabList gap="40px" w={{ base: "150%", md: "max-content" }} whiteSpace="nowrap" pl="4">
```

Comment:
I've tried it without the changes of these two lines and it still looks great to me. Is there any reason for keeping them?

Reply:
You right! They were probably related to previous attempts. I will make the changes now.